tools/generator/SourceWriters/Attributes/DebuggerBrowsableAttr.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.SourceWriter;
+
+namespace generator.SourceWriters
+{
+	public class DebuggerBrowsableAttr : AttributeWriter
+	{
+		public override void WriteAttribute (CodeWriter writer)
+		{
+			writer.WriteLine ("[DebuggerBrowsable (DebuggerBrowsableState.Never)]");
```

Comment:
I worry whenever we *don't* `global::…` fully qualify a type reference, as this has bitten us before.  (I'd swear there's a current instance where we need-to-but-don't emit `global:…`, but I can't find it.)